src/seedu/addressbook/commands/Command.java

```diff
@@ -36,6 +36,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
         return String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, personsDisplayed.size());
     }
 
+    public static String getMessageForPersonSortedListShownSummary(List<? extends ReadOnlyPerson> personsDisplayed) {
```

Comment:
This method declaration is rather long at 114 characters. Consider breaking the line into two. Refer to the style guide for more details:

![image](https://user-images.githubusercontent.com/35345446/52841171-81f7b380-3136-11e9-9977-e08484510f41.png)

Reply:
Noted!